src/Microsoft.ML.TimeSeries/PredictionFunction.cs

```diff
@@ -65,6 +65,13 @@ public sealed class TimeSeriesPredictionFunction<TSrc, TDst> : PredictionEngineB
         /// </summary>
         /// <param name="env">Usually <see cref="MLContext"/>.</param>
         /// <param name="modelPath">Path to file on disk where the updated model needs to be saved.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[CheckPoint](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries/DetectChangePointBySsa.cs)]
```

Comment:
>DetectChangePointBySsa [](start = 120, length = 22)

did you mean to reference the DetectIidSpike?

Reply:
no, I meant to referfence DetecthChangePointBySsa.

Reply:
I have added all of them,.